library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/aldebaran/110CU/Equality/aldebaran_Cijk_Ailk_Bljk_HHS_BH_Bias_AH_SDV_SAV.yaml

```diff
@@ -945,6 +945,7 @@
     ClusterLocalRead: 1
     CodeObjectVersion: V3
     # CustomKernelName: 'Custom_Cijk_Ailk_Bljk_HHS_BH_Bias_AH_SDV_SAV_MT32x32x64_MI16x16x16x1_SN_GRVWA8_GRVWB8_GSU7_K1_LBSPPA512_LPA16_LPB16_LRVW8_MIWT1_1_NTA0_NTB0_NEPBS2_SU0_SUS0_WG32_8_1_WGM1'
+    # BatchSizeEqual: 1
```

Comment:
If you don't want set BatchSizeEqual for this kernel, why don't you just remove "# BatchSizeEqual: 1"?